### DISCUSSION TITLE
Unoconv-context deadline exceeded

### DISCUSSION THREAD

**Discussion by IswaryaIyapalam** [gotenberg/gotenberg#408] [Q&A]
When i am hitting the libreoffice module, it is giving context deadline exceeded.  But the same file converted within 6 secs using demo.gotenberg.dev libreoffice api. When I increased process time to 7 mins, file is converted. But that is huge when compared to 5secs. How should i resolve the error? Need help very badly to reduce the conversion time

**Comment by gulien** [comment]
Are you using the latest version? If so, what’s your environment?

**Reply by IswaryaIyapalam** [reply]
Yes. I am using version 7.3. In linux environment, I am testing the conversions. Currently i dont have the server configuration. I will share it once i get it. and what is the recommended timeout for these three api-read-timeout, api-write-timout and api-process-timeout. Can it be extended from 10 mins till 1hr?

**Reply by gulien** [reply]
Consider upgrading to `7.4.2` (like the demo), where there is a new `--api-timeout` flag 😉  but yes, you can extend to 1 hour, it's up to you.

**Reply by IswaryaIyapalam** [reply]
I have upgraded to 7.4.2. But i am getting the same error in server. I have attached the log screenshot and server configuration. Is gotenberg uses any external service? I could see the file is converted in less than 30s without vpn connectivity in my local. What could be the possible reason?Is increasing the timeout value the only option? and what is the recommended system configuration?
![20220118_212629](https://user-images.githubusercontent.com/64399247/149971876-d7a283be-71f7-4721-a4f4-3351035a66b9.jpg)
![20220117_180928](https://user-images.githubusercontent.com/64399247/149971935-a6e2887f-2827-42dd-ab38-d4eb11ca3c32.jpg)

**Comment by gulien** [comment]
The demo environment has 1 CPU and 512 MB of RAM. If it's not working, you may try to:

* Increase the timeout with `--api-timeout`.
* Disable the unoconv listener with `--unoconv-disable-listener`.

See also #407.
